Vision/Vision.bs

```diff
@@ -107,77 +99,78 @@ Boilerplate: omit conformance
+	to provide a consistent architecture 
+	across the rapid pace of progress in the Web, 
+	and to build a common community to support its development. 
+	It has become an association where diverse voices from around the world 
+	and from different industries and organizations work together to evolve the Web.
+
+	To build a better future, the W3C must rise even further 
+	to the challenge of improving the Web's fundamental integrity, 
+	while continuing to expand the Web’s scope and reach. 
+	The Operational Principles for the W3C should reflect the core values of the Web itself.
+
+	These core values will be clearly demonstrated by how W3C leads the Web forward: 
+	by being inclusive, principled, and continually striving to make the Web better 
+	through these principles and the Ethical Web Principles. 
+	As the Ethical Web Principles state, “The web should empower an equitable, informed and interconnected society.”
+
+# Operational Principles for W3C # {#op-principles}
+
+	The fundamental function of W3C is to provide an open forum
 	where diverse voices from around the world
 	and from different organizations and industries
 	work together to build consensus
 	on voluntary global standards for Web technologies.
+	In order to fulfill this function, we will follow these operational principles:
 
-	* We put the needs of users first:
+	* W3C intentionally involves stakeholders from end to end 
+		in building the Web: developers, content creators, and end users. 
+		Our work will not be dominated by any person, company, or interest group, 
+		but uses consensus from the community to 
+		put the needs of users first: 
 		above authors, publishers, implementers, paying W3C Members, or theoretical purity.
-	* This organization believes in diversity
+	* We believe in diversity
 		and inclusion of participants from different
 		geographical locations,
 		cultures,
 		languages,
 		disabilities,
 		gender identities,
+		industries,
+		organizational sizes,
 		and more.
-	* We believe in principled, community-wide consensus-building
-		as the basis for building standards.
-	* We strongly emphasize accessibility,
+		In order to ensure the W3C serves the needs of the entire Web user base, 
+		we must also strive to broaden diversity and inclusion for our own participants.
+	* We create standards with an ethical intent to improve 
+		equity of access and participation on the Web. 
+		We will ensure the technical standards of the Web meets broad goals 
+		using consistent horizontal review to ensure accessibility, 
 		internationalization,
+		sustainability,
 		privacy,
 		and security.
-	* We believe the Web must be an environmentally sustainable platform,
-		as defined in the [[ETHICAL-WEB-PRINCIPLES inline|Ethical Web Principles]].
 	* Our standards are rooted in a strong royalty-free patent policy
-		and open copyright licenses,
-		openly developed with consensus of industry and key stakeholders.
+		and open copyright licenses.
```

Comment:
Note: We might want to say a bit about why, but maybe that's a separate issue. (The change looks fine.)